### PR TITLE
Output

### PR DESCRIPTION
**Pull request by jburel**
Adjust commit introduced in https://github.com/ome/omero-py/pull/39
https://github.com/ome/omero-py/pull/39/commits/f9a77bd416721caee745abe9038f610f5a9322a0
This broke the integration tests on CI
I was able to reproduce the issue locally too

this should fix https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-test-integration/194/testReport/junit/OmeroPy.test.integration.clitest.test_fs/TestFS/testRepos/
and
https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-test-integration/194/testReport/junit/OmeroPy.test.integration.clitest.test_fs/TestFS/testSetsWithTransfer/

cdff1dee505ce341ef6c8bd0dcb50ba5c7ac9801
should fix https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-test-integration/194/testReport/junit/OmeroPy.test.integration.clitest.test_user/TestUser/testList_None_last_name_/

### CODE REVIEW

File: src/omero/util/text.py [hunk 1]
```diff
@@ -320,7 +320,8 @@ def tostring(x):
                 try:
                     return str(x)
                 except UnicodeDecodeError:
-                    return '<Invalid UTF-8>'
+                    # Unicode characters are present
+                    return str(x.decode("utf-8", "ignore"))
```
Comment (joshmoore) [inline src/omero/util/text.py]: I'm have some concern that this is hiding a loss of information. e.g. someone queries, get's back `flgel` and then uses that as input. If automated, `'<Invalid UTF-8>'` had the same risk, but I'm unsure of what the original intent is.

Comment (jburel) [reply src/omero/util/text.py]: The output from the CLI command is actually ``Flügel``

Comment (manics) [reply src/omero/util/text.py]: What about if you use `replace` instead of `ignore`?

Comment (joshmoore) [reply src/omero/util/text.py]: > The output ... is actually Flügel

In other words, some previous bug has been remedied? If so, that's great. I wonder though if something else would get dropped.

Comment (jburel) [reply src/omero/util/text.py]: ``replace`` or ``ignore`` returns the same output for the case tested

Comment (manics) [reply src/omero/util/text.py]: Is this try-catch specifc to python 2 (I'm not too bothered since we know Python 2 has unicode issues) or Python 3 too?

Comment (jburel) [reply src/omero/util/text.py]: Python 2 there is another try-catch for Python 3

**Comment by joshmoore** [comment]
Only 3 unrelated integration tests left ([build 198](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-test-integration/198/#showFailuresLink)). Merging.

Note: if more discussion is needed on `ignore` vs. `replace` then an issue should likely be filed.

**Pull request merged**